package.json

```diff
@@ -345,7 +344,7 @@
                 "type": "object",
                 "description": "Destination for final Infoset ('file' | 'console' | 'none')",
                 "default": {
-                  "type": "console",
+                  "type": "file",
                   "path": "${workspaceFolder}/infoset.xml"
```

Comment:
The default location for this shoudl be "${workspaceFolder}/target/infoset.xml" so that it doesn't clutter and end up committed by accident.

Reply:
gotcha I like this idea. I am also still looking into the selecting a single JAR issue. It is working for me on Mac but you said it doesn't work for you (which I believe you use Linux) and believe @nlewis05 mentioned he saw that as well, possibly failing on windows as well. So I am currently getting up Linux VM up and running to get this fixed.

Reply:
@mbeckerle So after getting my ubuntu VM up and running I am able to get the same issue as you and @nlewis05 when it comes to picking out single JARs. It seems the VSCode API is limiting Linux so that you have to specify that the user can pick files OR folders and doesn't allow both. One way to get around it is when you open the file picker for the JARs bring up a selection box where you chose `Single File` or `Directory` before going to the file picker, this allow for us to set up the file picker properly. Or can you think of another, possibly better, way to get around it?

Reply:
Or maybe instead of having only the `Browse` button we make two buttons. One `Add Folder(s)` and one `Add File(s)` and then how is how the code will know how to show the file picker. I think this would be nicer then the other option I stated above.

Reply:
@mbeckerle @nlewis05 Please give this another look over. I made the daffodilDebugClasspath have two buttons like mentioned above to fix the issue.